applications/FluidDynamicsApplication/tests/couette_flow_test.py

```diff
@@ -0,0 +1,134 @@
+# Import Python modules
+import math
+
+# Import kratos core and applications
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as KratosUtilities
+import KratosMultiphysics.FluidDynamicsApplication as KratosFluid
+from KratosMultiphysics.FluidDynamicsApplication.fluid_dynamics_analysis import FluidDynamicsAnalysis
+
+class CouetteFlowTest(KratosUnittest.TestCase):
+    def testCouetteFlow2DSymbolicStokes(self):
+        self.solver_type = "MonolithicStokes"
+        self.element_type = "symbolic_stokes"
+        self.time_scheme = "bdf2"
+        self._CustomizeSimulationSettings()
+
+    def testCouetteFlow2DSymbolicNavierStokes(self):
+        self.solver_type = "Monolithic"
+        self.element_type = "symbolic"
+        self.time_scheme = "bdf2"
+        self._CustomizeSimulationSettings()
+
+    def setUp(self):
+        self.print_output = False
+        self.check_tolerance = 1.0e-6
+        self.print_reference_values = False
+        self.work_folder = "CouetteFlowTest"
+        settings_filename = "ProjectParameters.json"
+
+        # Read the simulation settings
+        with KratosUnittest.WorkFolderScope(self.work_folder,__file__):
+            with open(settings_filename,'r') as parameter_file:
+                self.parameters = KratosMultiphysics.Parameters(parameter_file.read())
+
+        # If required, add the output process
+        if self.print_output:
+            self._AddOutput()
+
+    def runTest(self):
+        # Create the test simulation
+        with KratosUnittest.WorkFolderScope(self.work_folder,__file__):
+            self.model = KratosMultiphysics.Model()
+            simulation = FluidDynamicsAnalysis(self.model, self.parameters)
+            simulation.Run()
+
+    def tearDown(self):
+        with KratosUnittest.WorkFolderScope(self.work_folder, __file__):
+            KratosUtilities.DeleteFileIfExisting('couette_flow_test.time')
+
+    def checkResults(self):
```

Comment:
why implementing this again?
This has a standard workflow in the core (using json files) that is extensively tested, is fast, works in MPI etc

Reply:
I did it similar to the other application tests. Makes sense to follow the core guidelines. I'll amend it.

Reply:
Concerning the mdpa addition, I can use the `StructuredMeshGenerationProcess` instead.

Reply:
JSON-based results check implemented.